web_tool_stress_test.R

```diff
@@ -71,6 +73,7 @@ company_exclusion <- if (exists("company_exclusion")) company_exclusion else FAL
 
 #### Analysis Parameters----------------------------------------
 # Get analysis parameters from the projects AnalysisParameters.yml - similar to PACTA_analysis
+# TODO: should this remain "working location"? comes from PACTA, so probably okay?
```

Comment:
yeah, I'm not sure... I guess we can cross that bridge when we get to it, but yeah, I had the same thought

Reply:
#9